### PR TITLE
AdditionalProperties description

### PR DESCRIPTION
**Pull request by ifeoluwaokunoren**
Updating the subscription lifecycle page to add the description for additional properties.

### CODE REVIEW

File: v1.0/subscription-lifecycle-api-reference.md [hunk 1]
```diff
@@ -23,43 +23,69 @@ This API uses the &quot;system&quot; version of 2.0 because it can be triggered
...
+		],
+		"subscriptionSpendingLimit": "True",
+		"subscriptionAccountOwner": "account@company.com",
+		"managedByTenants": [
+			{
+				"tenantId": "<managedByTenantId>"
+			}
+		],
+		"additionalProperties": {
+			"<key>": "<JToken1>",
```
Comment (laxmankumar12) [inline v1.0/subscription-lifecycle-api-reference.md]: key1 instead of just key

File: v1.0/subscription-lifecycle-api-reference.md [hunk 2]
```diff
@@ -23,43 +23,69 @@ This API uses the &quot;system&quot; version of 2.0 because it can be triggered
...
+			"<keyN>": "<JTokenN>"
+		},
+		"resourceProviderProperties": {
```
Comment (laxmankumar12) [inline v1.0/subscription-lifecycle-api-reference.md]: This should be inside additionalProperties.

File: v1.0/subscription-lifecycle-api-reference.md [hunk 3]
```diff
@@ -23,43 +23,69 @@ This API uses the &quot;system&quot; version of 2.0 because it can be triggered
...
-| **Properties.managedByTenants** | Optional.All tenants managing the subscription. Null or empty means that there are no managing tenants. |
+| **properties.registeredFeatures** | Optional.All AFEC features that the subscriptions has been registered under RP namespace and platform namespace (Microsoft.Resources).  Null or an empty array would mean that there are no registered features in the subscription. | **Properties.availabilityZones** | Optional.Physical to logical zone mapping (need to add above).  it will be per-region.|
+| **properties.managedByTenants** | Optional.All tenants managing the subscription. Null or empty means that there are no managing tenants. |
+| **properties.additionalProperties** | Required. AdditionalProperty bag, which is a dictionary of JTokens. More details can be found below. | 
+| **properties.resourceProviderNamespace**| Required. Resource Provider Namespace e.g.: Microsoft.Contonso. |
```
Comment (laxmankumar12) [inline v1.0/subscription-lifecycle-api-reference.md]: This should be similar to billingProperties. It is inside additionalProperties.

**Review by laxmankumar12 — APPROVED** [review]

File: v1.0/subscription-lifecycle-api-reference.md [hunk 4]
```diff
@@ -23,43 +23,78 @@ This API uses the &quot;system&quot; version of 2.0 because it can be triggered
...
+		"resourceProviderProperties": {
+			"resourceProviderNamespace": "<Provider Namespace>"
+		}
+	}
+}
+```
+| **Element name** | Description |
+| --- | --- |
+| **additionalproperties.billingproperties**| <placeholder> |
+| **billingproperties.costCategory**| <placeholder> |
```
Comment (perseusCode) [inline v1.0/subscription-lifecycle-api-reference.md]: What are we adding in placeholder ?

**Review by perseusCode — APPROVED** [review]

**Pull request merged**